docs/source/manifest_submission.rst

```diff
@@ -0,0 +1,274 @@
+Submit a manifest to Synapse
+============================
+
+Prerequisites
+-------------
+
+**Obtain Synapse Credentials**:
+Ensure you have a Synapse account and have obtained your Synapse credential by following the instructions here:
+    `<https://python-docs.synapse.org/tutorials/authentication/#prerequisites>`_
+
+**Before Using the Schematic CLI**
+
+- **Install and Configure Schematic**:
+  Ensure you have installed `schematic` and set up its dependencies.
+  Refer to the **"Installation Guide for Users"** for detailed instructions.
+
+- **Understand Important Concepts**:
+  Familiarize yourself with key concepts outlined on the :doc:`index.rst` page.
+
+- **Configuration File**:
+  Learn more about each attribute in the configuration file by referring to the relevant documentation.
+
+- **Obtain a manifest**:
+  Please obtain a manifest by following the documentation of generating a manifest.
+
```

Comment:
Should we link to the generating a manifest doc here?

Reply:
there is.. but it is hard to do it in this PR because the page of manifest generation does not exist. I will commit to Tom's branch once both PR get merged